daemon/seccomp_linux_test.go

```diff
@@ -142,7 +145,7 @@ func TestWithSeccomp(t *testing.T) {
 			comment: "load daemon's profile",
 			daemon: &Daemon{
 				seccompEnabled: true,
-				seccompProfile: []byte("{ \"defaultAction\": \"SCMP_ACT_ERRNO\" }"),
+				seccompProfile: []byte("{ \"defaultAction\": \"SCMP_ACT_ERRNO\", \"defaultErrnoRet\": 1 }"),
```

Comment:
Probably not for this PR, but we should consider using string literals (back ticks) in this test, so that we don't have to do all the escaping for the quotes 😞

Reply:
heh, that is what I thought when doing the change. I can move it to in this PR or a different one, if you want :)

Reply:
Yeah, either way it would cause some code-churn, but I definitely think it makes the test-cases more readable. Perhaps as an extra commit in this PR?

In that case: thinking what would be most useful; perhaps do the backticks change as first commit, so that the "last visible change"  ("git blame") shows the new addition.

(and then 🤞 none of the other reviewers will complain "why make that change?" 😅 🤗)

Reply:
Will add it as a second commit, in case some reviewer wants to revert it so it is simple. It is slightly more work in case of a git blame, but that is likely to be the case as this mutates too.

Let me know if you think as a first commit is better and I can adapt it :)